src/internal/register.ts

```diff
@@ -16,6 +16,15 @@ export interface RegisterConfigs {
   audiences?: Audience[];
 }
 
+//
+// fyi, the init flag is because we can't await pipeline initialization
+// in our constructor so we do it on processPipeline only if needed
+export const internal = {
+  pipelineInitialized: false,
+  pipelines: new Map<string, InternalPipeline>(),
```

Comment:
JS allows object as map and set keys...but they are stored by reference so you can only fetch them by reference...so stringifying the keys.